File: payola/web/client/src/main/scala/cz/payola/web/client/RpcTestClient.scala
package cz.payola.web.client

import s2js.adapters.js.browser._
import cz.payola.web.shared.RpcTestRemote
import cz.payola.common.rdf.{Graph}

object RpcTestClient
{
    def test() {
        window.alert(RpcTestRemote.foo(123, "xyz"))
    }

    def getRandomGraph:Graph = {
        RpcTestRemote.getRandomGraph
    }
}

File: payola/web/server/public/javascripts/client/cz/payola/web/client/RpcTestClient.js
goog.provide('cz.payola.web.client.RpcTestClient');
goog.require('s2js.Rpc');
cz.payola.web.client.RpcTestClient.test = function() {
var self = this;
window.alert(s2js.Rpc.callSync('cz.payola.web.shared.RpcTestRemote.foo',[123, 'xyz']));
};
cz.payola.web.client.RpcTestClient.metaClass_ = new s2js.MetaClass('cz.payola.web.client.RpcTestClient', []);
